astor/misc.py

```diff
@@ -170,6 +177,23 @@ def parsefile(fname):
     return ast.parse(fstr, filename=fname)
 
 
+def finfo(codeobj):
```

Comment:
We are not very consistent about naming things (e.g. parsefile instead of parse_file), but we can name this `get_filename` to be more PEP 8 friendly.